### ISSUE TITLE
Switch from `--heroku` flag to `--no-heroku` flag

### ISSUE THREAD

**Issue by croaky** [thoughtbot/suspenders#510]
We've heard from a number of folks at thoughtbot that they forget to use the `--heroku` flag and then they need to manually add what Suspenders does for Heroku config.

**Comment by gylaz** [comment]
:+1:

**Comment by teoljungberg** [comment]
Makes sense, the anomaly would be to not use heroku in our case

**Comment by tute** [comment]
I'd wait on https://github.com/thoughtbot/suspenders/pull/511#issuecomment-72885271, that will handle this case. Should we close this issue in favor of that PR?

**Comment by croaky** [comment]
Yup, sounds good.

**Comment by tute** [comment]
Closing in favor of https://github.com/thoughtbot/suspenders/pull/511.
